internal/common/model/message/ingress.go

```diff
@@ -8,6 +8,7 @@ type IngressValidationRequest struct {
 	Metadata    map[string]string `json:"metadata"`
 	RequestID   string            `json:"request_id"`
 	Principal   string            `json:"principal"`
+	OrgID       string            `json:"org_id"`
 	Service     string            `json:"service"`
```

Comment:
Question: Do you happen to know if ingress is sending anything under the `Principal` field? We can probably remove it, since it doesn't look like we are using it anywhere in `response_consumer` (worth verifying).

Reply:
Yeah, ingress is sending `OrgID` under the `principal` field in the messages on the `announce` topic. `principal` is no longer being used in playbook dispatcher after this change anywhere, so yeah we could def remove it. Do you want to do that as a part of this ticket or make a new one?